biz.aQute.bndlib/bnd.bnd

```diff
@@ -73,6 +75,7 @@ Import-Package: \
 	org.osgi.service.repository;version=latest,\
 	org.osgi.util.function;version=latest,\
 	org.osgi.util.promise;version=latest,\
+	org.osgi.dto;version=latest,\
```

Comment:
osgi.core (above) already contains the dto package. So you do not need to add org.osgi.dto to the buildpath.

Reply:
I think you deleted the wrong line. You need org.osgi.dto in the Import-Package statement. You do not need org.osgi.dto in the -buildpath instruction since osgi.core is already in the -buildpath instruction and it contains the org.osgi.dto package.